packages/runtime/datastore/src/remoteChannelContext.ts

```diff
@@ -68,6 +68,10 @@ export class RemoteChannelContext implements IChannelContext {
         getBaseGCDetails: () => Promise<IGarbageCollectionDetailsBase>,
         private readonly attachMessageType?: string,
     ) {
+        if (this.id.includes("/")) {
+            throw new UsageError("Ids for LocalChannelContextBase cannot have '/' characters");
```

Comment:
Similar to my comments for remote data store context, this should be an assert instead of usage error.